store/copr/mpp.go

```diff
@@ -150,7 +152,10 @@ func (m *mppIterator) run(ctx context.Context) {
 		m.mu.Unlock()
 		m.wg.Add(1)
 		bo := backoff.NewBackoffer(ctx, copNextMaxBackoff)
-		go m.handleDispatchReq(ctx, bo, task)
+		go func(mppTask *kv.MPPDispatchRequest) {
+			m.handleDispatchReq(ctx, bo, mppTask)
+			m.wg.Done()
```

Comment:
+1 for should better `defer` to avoid unknown panic in future

Reply:
done